scripts/proto.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/python
+
+from os import pathsep, defpath, environ, walk, system
```

Comment:
Use ``os.walk``, ``os.system`` etc rather than importing function for readability.